app/views/survey/demographic_questions/_wwf_programme.html.erb

```diff
@@ -1,28 +0,0 @@
-<p class="card-text">
```

Comment:
Something seems to have gone a bit wrong here with the commits. There's a few duplicates as in the last PR

Reply:
Sorry I did notice that but was not sure why that would have happened.

Reply:
Something was wrong with the way I created a new branch which meant other changes carried over which were meant to be in a separate branch.

Reply:
Ah I think you branched off the previous branch instead of master. There would be no harm in putting these all in the same PR too as they're all related to one feature, but not to worry. You just need to figure out how to get this branch in line with master before I can merge it